### PR TITLE
[UR] [CUDA] Changed the output of querying localMemSize

### PR DESCRIPTION
**Pull request by konradkusiak97**
CI for https://github.com/oneapi-src/unified-runtime/pull/931

### CODE REVIEW

File: sycl/plugins/unified_runtime/CMakeLists.txt [hunk 1]
```diff
@@ -53,14 +53,15 @@ endif()
...
+  # !! Do not merge
+  set(UNIFIED_RUNTIME_REPO "https://github.com/konradkusiak97/unified-runtime.git")
   # commit ba994bb401aefad3631012d592be672f6ea0e23b
   # Merge: be53fb3b f2a365ca
   # Author: Kenneth Benzie (Benie) <k.benzie@codeplay.com>
   # Date:   Fri Nov 3 11:09:52 2023 +0000
   #     Merge pull request #968 from martygrant/martin/versionCheckSpecializationConstants
   #     [OpenCL] Add version check for urProgramSetSpecializationConstants
-  set(UNIFIED_RUNTIME_TAG ba994bb401aefad3631012d592be672f6ea0e23b)
+  set(UNIFIED_RUNTIME_TAG konradkusiak/LocalMemSizeQuery)
```
Comment (kbenzie) [inline sycl/plugins/unified_runtime/CMakeLists.txt:64]:
```suggestion
  set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
  # commit 0e24ab8d136978d984be9e8cba34e23b4fbcf37c
  # Merge: a62423d d5a4691
  # Author: Kenneth Benzie (Benie) <k.benzie@codeplay.com>
  # Date:   Mon Nov 6 11:42:03 2023 +0000
  #     Merge pull request #931 from konradkusiak97/konradkusiak/LocalMemSizeQuery
  #     [UR] [CUDA] Changed the output of querying localMemSize
  set(UNIFIED_RUNTIME_TAG 0e24ab8d136978d984be9e8cba34e23b4fbcf37c)
```

**Review by kbenzie — CHANGES_REQUESTED** [review]
Now that https://github.com/oneapi-src/unified-runtime/pull/931 is merged please merge the latest sycl branch changes and update `UNIFIED_RUNTIME_REPO`/`UNIFIED_RUNTIME_TAG` as shown below. Then mark this PR as ready for review.

File: sycl/plugins/unified_runtime/CMakeLists.txt (hunk 1 above)
Comment (konradkusiak97) [reply sycl/plugins/unified_runtime/CMakeLists.txt]: OK, updated

**Review by kbenzie — APPROVED** [review]
UR LGTM

**Comment by kbenzie** [comment]
@intel/llvm-gatekeepers this is ready to merge.

**Pull request merged**